backend/persistent/QueryScheduler.js

```diff
@@ -1,11 +1,23 @@
-import * as Connections from './datastores/Datastores';
-import {getConnectionById} from './Connections';
-import {getQuery, getQueries, saveQuery, deleteQuery} from './Queries';
-import {getSetting} from '../settings';
-import Logger from '../logger';
-import {PlotlyAPIRequest, updateGrid} from './plotly-api.js';
 import {has} from 'ramda';
 
+import {getConnectionById} from './Connections.js';
+import * as Connections from './datastores/Datastores.js';
+import Logger from '../logger';
+import {
+    getQuery,
+    getQueries,
+    saveQuery,
+    deleteQuery
+} from './Queries.js';
```

Comment:
(Not necessarily in this PR) but we should small-case these filenames as well.. !! `Queries`, `Datastore` etc, and remove `.js` extensions to be consistent.. !!

Reply:
I want to keep the extensions, because:
- it's grep-friendly
- we have imports that only differ in the extension (e.g. `code-editor.css` and `code-editor.jsx`)

Reply:
The idea is to move to lower case, whenever we have a chance (instead of having a huge big PR). We don't really want to spend time on it.